packages/pn-personafisica-webapp/src/redux/notification/actions.ts

```diff
@@ -115,15 +121,50 @@ export const getNotificationPaymentInfo = createAsyncThunk<
   ) => {
     try {
       const { notificationState } = getState();
+      const paymentCache = getPaymentCache();
+
+      if (
```

Comment:
if we move all the validation checks into the getPaymentCache method, this if condition can be replaced with if(paymentCache) ...